src/qgitrepository.cpp

```diff
@@ -624,4 +627,17 @@ bool Repository::shouldIgnore(const QString &path) const
     return result;
 }
 
+void Repository::setIdent(const QString &name, const QString &email)
+{
+    qGitThrow(git_repository_set_ident(data(), name.toLatin1(), email.toLatin1()));
+}
+
+Repository::Ident Repository::ident() const
+{
+    const char *name;
+    const char *email;
+    qGitThrow(git_repository_ident(&name, &email, data()));
+    return Ident{ QString::fromLatin1(name), QString::fromLatin1(email) };
```

Comment:
I think `QString::toUtf8()` and `QString::fromUtf8()` should be used here. Those are used in `Signature` class.

Reply:
Fixed.